### DISCUSSION TITLE
Error code "stem not included in save location.  May cause overwrite"

### DISCUSSION THREAD

**Discussion by culturedealer** [CarlGao4/Demucs-Gui#55] [Q&A] [answered]
Sorry if this is a simple issue.

I was using demucs last week without this error - tried to update and can't figure out what happened or how to restore.  This comes up whenever I try to start separation and it gets stuck on "writing."  I realize its likely simple but any help would be appreciated.

**Comment by CarlGao4** [comment, marked as answer]
This is a bug introduced in 1.1b2, which will use the last output file name as the default save location. Once you've updated to 1.1, Please click on the menu (about), then click "Clear saved file location", and restart the app.

**Reply by culturedealer** [reply]
Many Thanks.  Perfect.
